static/css/SCSS/home_page.scss

```diff
@@ -0,0 +1,382 @@
+@import url(font-awesome.min.css);
```

Comment:
is this resolving to http? because we ship with font-awesome, no need to import from web